src/components/ae-address/ae-address.vue

```diff
@@ -68,7 +68,7 @@ export default {
      * @return {String[]}
      */
     chunked() {
-      return this.value.match(/.{1,3}/g);
+      return this.value.match(/^(\w{2}_)|.{2}(?=.{47,53})|.{2,3}/g);
```

Comment:
Is it intentional to split `ak_QY8VNEkhj7omMUjAvfVBq2NjTDy895LBYbk7qVxQo1qT8VqfE1` as `ak_ QY 8VN Ekh j7o mMU jAv fVB q2N jTD y89 5LB Ybk 7qV xQo 1qT 8Vq fE1`?
Brakets around `\w{2}_` looks useless.

Reply:
@davidyuk 

Yes, generally we want to have:

- `ak_` as 3 characters at the beginning
- first two characters
- end of the address needs to display either 2 or 3 chars

This is how its generally in the designs.

Reply:
@davidyuk I checked the tests, and there seem to be an issue:

We need to group into a pair of two the first two characters after `ak_` this is based on the designs which wont change I think.

Reply:
I have not seen this screen:
https://app.zeplin.io/project/59e4cf99fc0bb2f99a89551b/screen/5bd1cec7e9f15622d2d99784
So looks like it is the intentioned behavior.